src/broker/broker_log_util.c

```diff
@@ -209,27 +222,24 @@ is_bind_with_size (char *buf, int *tot_val_size, int *info_size)
       *info_size = (char *) info_end - (char *) buf;
     }
 
-  if ((strncmp (p, "CHAR", 4) != 0) || (strncmp (p, "VARCHAR", 7) != 0) || (strncmp (p, "NCHAR", 5) != 0)
-      || (strncmp (p, "VARNCHAR", 8) != 0))
-    {
-      *tot_val_size = strlen (info_end);
-    }
-  else if (tot_val_size)
+
+  if (tot_val_size)
     {
-      len = size_end - size_begin;
-      if (len > (int) sizeof (size))
-	{
-	  goto error_on_val_size;
-	}
-      if (len > 0)
+      if (memcmp (p, "BIT", 3) && memcmp (p, "VARBIT", 6))
```

Comment:
해당 if문 조건 한 번 확인 부탁드립니다.
```suggestion
      if (memcmp (p, "BIT", 3) || memcmp (p, "VARBIT", 6))
```

Reply:
BIT 또는 VARBIT 라면 238라인으로 진행하기 위한 코드입니다.
제안 주신 내용처럼 수정하면 해당 if문은 항상 true 가 될 것입니다.
`memcmp (p, "BIT", 3)` 과 `memcmp (p, "VARBIT", 6)` 중에 한쪽은 반드시 `0` 이 아닌 값이 나올 것이기 때문에요

Reply:
답변 감사합니다.
memcmp (p, "BIT", 3) != 0 && memcmp (p, "VARBIT", 6) != 0 를 의미하신 거였군요.